controller/src/main/java/io/pravega/controller/store/stream/PersistentStreamBase.java

```diff
@@ -1254,34 +1290,47 @@ private boolean isStreamCutValidInternal(Map<Long, Long> streamCut, int epochLow
                                     .getSegments().stream()
                                     .forEach(x -> duplicateActiveSegmentsBuilder.add(
                                             newSegmentRecord(computeSegmentId(getSegmentNumber(x.segmentId()),
-                                            committingTxnRecord.getNewActiveEpoch()),
-                                            timeStamp + 1, x.getKeyStart(), x.getKeyEnd())));
-
-                            EpochRecord duplicateTxnEpoch = new EpochRecord(committingTxnRecord.getNewTxnEpoch(), 
-                                    transactionEpochRecord.getReferenceEpoch(), duplicateTxnSegmentsBuilder.build(), timeStamp);
-
-                            EpochRecord duplicateActiveEpoch = new EpochRecord(committingTxnRecord.getNewActiveEpoch(),
-                                    activeEpochRecord.getReferenceEpoch(), duplicateActiveSegmentsBuilder.build(), timeStamp + 1);
-
-                            HistoryTimeSeriesRecord timeSeriesRecordTxnEpoch =
-                                    new HistoryTimeSeriesRecord(duplicateTxnEpoch.getEpoch(), duplicateTxnEpoch.getReferenceEpoch(), 
-                                            ImmutableList.of(), ImmutableList.of(), timeStamp);
-
-                            HistoryTimeSeriesRecord timeSeriesRecordActiveEpoch =
-                                    new HistoryTimeSeriesRecord(duplicateActiveEpoch.getEpoch(), duplicateActiveEpoch.getReferenceEpoch(),
-                                            ImmutableList.of(), ImmutableList.of(), timeStamp + 1);
-                            return createEpochRecord(duplicateTxnEpoch)
-                                    .thenCompose(x -> updateHistoryTimeSeries(timeSeriesRecordTxnEpoch))
-                                    .thenCompose(x -> createEpochRecord(duplicateActiveEpoch))
-                                    .thenCompose(x -> updateHistoryTimeSeries(timeSeriesRecordActiveEpoch))
-                                    .thenCompose(x -> createSegmentSealedEpochRecords(
-                                            activeEpochRecord.getSegments().stream().map(StreamSegmentRecord::segmentId)
-                                                             .collect(Collectors.toList()), duplicateTxnEpoch.getEpoch()))
-                                    .thenCompose(x -> createSegmentSealedEpochRecords(
-                                            duplicateTxnEpoch.getSegments().stream().map(StreamSegmentRecord::segmentId)
-                                                             .collect(Collectors.toList()), duplicateActiveEpoch.getEpoch()));
-                        })
-                        .thenCompose(r -> updateSealedSegmentSizes(sealedTxnEpochSegments)));
+                                                    committingTxnRecord.getNewActiveEpoch()),
```

Comment:
i dont see any code change in this block. could you please revert this. thanks

Reply:
fixed.